scripts/is/intg/helper-scripts/add_patch_repository.sh

```
@@ -0,0 +1,113 @@
+    ADDED=false
+    while IFS= read -r line; do
+        echo "$line" >> "$TEMP_FILE"
+        if [[ "$line" == *"<scm>"* ]] && [ "$ADDED" = false ]; then
+            echo "    <repositories>" >> "$TEMP_FILE"
+            echo "$WSO2_REPO" >> "$TEMP_FILE"
+            echo "    </repositories>" >> "$TEMP_FILE"
+            ADDED=true
+        elif [[ "$line" == *"</project>"* ]] && [ "$ADDED" = false ]; then
+            # Insert before the closing project tag
+            sed -i '' '$d' "$TEMP_FILE" # Remove the last line which is </project>
+            echo "    <repositories>" >> "$TEMP_FILE"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)